src/app/client/src/app/modules/core/components/main-footer/main-footer.component.ts

```diff
@@ -102,4 +116,15 @@ footerAlign() {
     };
   }
 
+  getTenantConfig() {
+    const slug = this.userService.slug ? this.userService.slug : this.defaultTenant;
+    this.tenantService.getSlugDefaultTenantInfo(slug, this.defaultTenant).pipe(takeUntil(this.unsubscribe$)).subscribe(
+      (configResponse) => {
+        this.tenantFooter.helpCenterLink = _.get(configResponse, 'helpCenterLink');
```

Comment:
do not handpick the objects/keys from the response object.  Define an object called footer in the settings & take the complete object. So later we may update any additional keys into init, doesn't require code change here

Reply:
- Please provide the sample json structure; so that it can be mapped in html (Bindings)